Office Add-ins community call (monthly).ics

```diff
@@ -0,0 +1,308 @@
+BEGIN:VCALENDAR
+PRODID:-//Microsoft Corporation//Outlook 16.0 MIMEDIR//EN
+VERSION:2.0
+METHOD:REQUEST
+X-MS-OLK-FORCEINSPECTOROPEN:TRUE
+BEGIN:VTIMEZONE
+TZID:China Standard Time
+BEGIN:STANDARD
+DTSTART:16010101T000000
+TZOFFSETFROM:+0800
+TZOFFSETTO:+0800
+END:STANDARD
+END:VTIMEZONE
+BEGIN:VEVENT
+ATTENDEE;CN="Office Platform PM - Internal";RSVP=FALSE:mailto:apiextpmi@mic
+	rosoft.com
+ATTENDEE;CN="Calc China PM & Design";RSVP=FALSE:mailto:modcsecopd@microsoft
+	.com
+ATTENDEE;CN="Anjali T P";ROLE=OPT-PARTICIPANT;RSVP=FALSE:mailto:Anjali.T@mi
+	crosoft.com
+ATTENDEE;CN="Rajesh Kumar Jha";ROLE=OPT-PARTICIPANT;RSVP=FALSE:mailto:Rajes
+	hKumar.Jha@microsoft.com
+CLASS:PUBLIC
+CREATED:20250208T055544Z
+DESCRIPTION:Go to this link to join the meeting: https://aka.ms/officeaddin
+	scall-join <https://nam06.safelinks.protection.outlook.com/?url=https%3A%2
+	F%2Faka.ms%2Fofficeaddinscall-join&data=05%7C02%7Cmingjialiu%40microsoft.c
+	om%7C6348f16cc14b41cb566408dc002d6aa5%7C72f988bf86f141af91ab2d7cd011db47%7
+	C1%7C0%7C638385442381031256%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLC
+	JQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=xoycnKL
+	LDuWH6B4BL3d%2FmDDyRcgRbJ%2BFVb0Wv660Njs%3D&reserved=0> \n\n \n\nThe Offic
+	e Add-ins community call is our monthly event for developers to connect in
+	 real-time with the folks who are building the Office JS platform and APIs
+	. During each call\, we share updates related to the Office JS platform an
+	d APIs and provide tips about useful tools\, samples\, and resources for d
+	eveloping Office Add-ins. These calls provide us with an excellent opportu
+	nity to connect directly with our customers and I hope you’ll take the t
+	ime to join us.\n\n \n\nTime details\n\nThe call is from 7am – 8am PST. 
+	This invite reserves 15 minutes pre-call for setup. If you’re presenting

```

Comment:
The description and attendees seem to indicate this is the invite for Microsoft employees. If we are posting this in a public repo, we should make the text targeted to developers. Could we copy the descriptions from the old invite, or is this change intentional?